gemrb/GUIScripts/pst/GUIREC.py

```diff
@@ -838,7 +832,13 @@ def AcceptLevelUp():
 			GemRB.SetPlayerStat (pc, IE_LEVEL2, GemRB.GetPlayerStat (pc, IE_LEVEL2)+NumOfSecoLevUp)
 	
 	LUSkillsSelection.SkillsSave (pc)
-	UpdateRecordsWindow ()
+
+	GUICommonWindows.OptionsWindow.SetVisible (True)
+	GUICommonWindows.PortraitWindow.SetVisible (True)
+	GUICommonWindows.ActionsWindow.SetVisible (True)
+	RecordsWindow.SetVisible (True)
+
+	UpdateRecordsWindow (RecordsWindow)
```

Comment:
You wouldn't need to manually call `UpdateRecordsWindow` if you set `UpdateRecordsWindow` as a handler for  the `ACTION_WINDOW_FOCUS_GAINED` action

Reply:
Hmm, I see that should be covered by CreateTopWinLoader, so perhaps this part is redundant? I think I just patched it from what was already there, to avoid a python error.

Reply:
That doesn't appear to apply here because `OpenLevelUpWindow` doesn't use that mechanism (tho it probably should).